install/configuring_inventory_file.adoc

```diff
@@ -1316,11 +1316,24 @@ the Ansible host file at install:
 
 ----
 [OSEv3:vars]
-os_firewall_use_firewalld=True
+os_firewall_use_firewalld=True <1>
 ----
+<1> Setting this variable to `true` opens the required ports and adds rules to
+the default zone, ensuring that firewalld is configured correctly.
+
+[IMPORTANT]
+====
+*firewalld* is not supported on Atomic Host. But you can use it by supplying an additional `openshift_enable_unsupported_configurations` parameter.
```

Comment:
s/Host. But/Host, but
s/parameter./parameter in the Ansible host file:  (or other correct file)

Reply:
I don't think we should mention the unsupported options here.